documentation/Get-PnPRecycleBinItem.md

```diff
@@ -39,11 +40,13 @@ Get-PnPRecycleBinItem [-SecondStage] [-RowLimit <Int32>] [-Connection <PnPConnec
 ```
 
 ## DESCRIPTION
+This command will return all the items in the recycle bin for the SharePoint site you connected to with Connect-PnPOnline. You must connect as a Site Collection Owner or Administrator. The SharePoint Admin Role in the tenant alone will not work. If you are not a Site Collection Admin connect to the Tenant Admin URL with Connect-PnPOnline and use Get-PnPTenantRecycleBinItem. 
 
 ## EXAMPLES
 
 ### EXAMPLE 1
 ```powershell
+Connect-PnPOnline -Url https://contoso.sharepoint.com/sites/askhr
```

Comment:
This line is not really that needed I think, as without it PnP PowerShell will not work. Otherwise we would have to update all the documention for every cmdlet.  I think it's quite safe to assume people know of the existance of Connect-PnPOnline. Also when they try the cmdlet without Connect-PnPOnline it will tell you to use Connect-PnPOnline first.

Reply:
I added that to show that when you use this command (as opposed to the "tenant" version) that you connect directly to the site collection itself, not the Admin site. I understand if you take it out.